test/misc/bench.ts

```diff
@@ -16,14 +16,13 @@
 
 
 const v8debugapi = require('../src/v8debugapi.js');
-const Logger = require('../src/logger.js');
 const config = require('../config.js').default;
 import * as assert from 'assert';
 const pretty = require('pretty-hrtime');
-//const util = require('util');
+// const util = require('util');
 const fib = require('./bench-code.js');
 
-const logger = new Logger(config.logLevel);
+const logger = new (require('../src/logger.js'))(config.logLevel);
```

Comment:
Why is this require call being inlined here?

Reply:
This was just to follow the style guide.  The use of `const Logger = ...` is an error since the name starts with a uppercase letter with the rest lowercase.

Reply:
In that case, use an inline tslint disable comment to disable that one line rather than moving the require.

Reply:
Done